code/modules/hazmat/crystal.dm

```diff
@@ -0,0 +1,42 @@
+#define CRYSTAL_SHAPE_SQUARE "square"
+#define CRYSTAL_SHAPE_CIRCLE "circle"
+#define CRYSTAL_SHAPE_TRIANGLE "triangle"
+#define CRYSTAL_SHAPE_CYLINDER "cylinder"
+
+#define CRYSTAL_SIZE_SMALL "small"
+#define CRYSTAL_SIZE_MEDIUM "medium"
+#define CRYSTAL_SIZE_LARGE "large"
+
+#define CRYSTAL_COLOR_RED "#FF0000"
+#define CRYSTAL_COLOR_BLUE "#00FF00"
+#define CRYSTAL_COLOR_GREEN "#0000FF
```

Comment:
If you're curious why it doesn't compile

Reply:
remember that aliens colonial marines typo that made the xenos dumb?
```suggestion
#define CRYSTAL_COLOR_GREEN "#0000FF"
```